components/automate-gateway/api/iam/v2/teams.proto

```diff
@@ -4,14 +4,75 @@ package chef.automate.api.iam.v2;
 option go_package = "github.com/chef/automate/components/automate-gateway/api/iam/v2";
 
 import "google/api/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";
 
 import "components/automate-gateway/api/iam/v2/request/teams.proto";
 import "components/automate-gateway/api/iam/v2/response/teams.proto";
 import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
 import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service Teams {
+  /*
+  Create a local team
+
+  Creates a local team, used to group local users as members of IAM policies.
+
+  Authorization Action:
+  ```
+  iam:teams:create
+  ```
+  */
+  rpc CreateTeam (CreateTeamReq) returns (CreateTeamResp) {
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "teams";
+			extensions: {
+				key: "x-code-samples";
+				value {
+					list_value: {
+						values: {
+							struct_value {
+								fields {
+									key: "lang";
+									value {
+										string_value: "JSON";
+									}
+								}
+								fields {
+									key: "source";
+									value {
+                    string_value: '{"id": "test-id", "name": "My Test Team", "projects": ["project1", "project2"]}';
+                  }
+								}
+							}
+						}
+					}
+				}
+			}
+		};
+    option (google.api.http) = {
+      post: "/iam/v2/teams"
+      body: "*"
+    };
+    option (chef.automate.api.policy).resource = "auth:teams";
+    option (chef.automate.api.policy).action = "create";
+    option (chef.automate.api.iam.policy).resource = "iam:teams";
+    option (chef.automate.api.iam.policy).action = "iam:teams:create";
+  };
+
+  /*
+  List all local teams
+
+  List all local teams.
```

Comment:
Tense is inconsistent throughout the file: here we "**list** all locals teams" but GetTeam "**returns** the details"
(@chef/docs-team probably has an opinion about which we want)

Reply:
Looks like the verdict is 

> The docs opinion is "active voice & present tense".

So I think List, Delete, Update, and at least a few others need to add an `s`